package.json

```diff
@@ -2,7 +2,7 @@
   "name": "px-file-upload",
   "author": "General Electric",
   "description": "A drag and drop file upload component.",
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Comment:
Unless you're a repo maintainer, it's better to leave version bumps to whomever releases your change.

The design system follows [semantic versioning](https://semver.org/#summary) when creating releases, in this instance a minor bump isn't correct, it should be a patch (as no functionality has been added).